spec/models/comment_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+describe Comment do
+  context "Validate" do
+    it 'should have a valid factory' do
+      FactoryGirl.create(:comment)
```

Comment:
@n1cx yang ini gw ikutin contoh yang di article nicho..
 okay noted", i will fix it